robowflex_ompl/src/ompl_interface.cpp

```diff
@@ -50,7 +50,7 @@ planning_interface::MotionPlanResponse OMPL::OMPLInterfacePlanner::plan(
     const SceneConstPtr &scene, const planning_interface::MotionPlanRequest &request)
 {
     planning_interface::MotionPlanResponse response;
-    response.error_code_.val = moveit_msgs::MoveItErrorCodes::FAILURE;
+    response.error_code_.val = moveit_msgs::MoveItErrorCodes::SUCCESS;
```

Comment:
Does this instead always return success even if the plan fails or is this now the correct behavior?

Reply:
When the plan fails it returns PLANNING_FAILED. See [here: ](uhttps://github.com/ros-planning/moveit/blob/master/moveit_planners/ompl/ompl_interface src/model_based_planning_context.cpp#L565rl). It only sets the error code when it fails.  However when the context fails It will return success. There are 2 ways to address this. Either use the other getPlanningContextFunction [here:]( http://docs.ros.org/indigo/api/moveit_core/html/classplanning__interface_1_1PlannerManager.html) . Or manually set FAILURE when getting the Context Fails.